installation/deploy.md

```diff
@@ -0,0 +1,22 @@
+# Prod environment deployment guide
+
+Note: prod deployment is very manual and not automated yet.
+
+- Login to the off1 server, as the "off" user
+- cd /home/off/openfoodfacts-server
+- Check that you are on the main branch
+- git pull
+- Copy changed files (don't copy everything, in particular not the lang directory that is being moved to the openfoodfacts-web repository)
+- e.g. cp cgi scripts lib po taxonomies templates /srv/off/
```

Comment:
would be could to have this command as an rsync in a small .sh file (target being $1)